volatility3/framework/symbols/windows/extensions/mft.py

```diff
@@ -10,15 +10,15 @@
 class MFTEntry(objects.StructType):
     """This represents the base MFT Record"""
 
-    def get_signature(self) -> str:
+    def get_signature(self) -> "objects.String":
```

Comment:
```suggestion
    def get_signature(self) -> objects.String:
```
Does importing the objects class cause a circular reference here? If not, I'd ditch the quotes, they're usually used to delay requirements to avoid needing to import classes, but it looks like the class has already been imported by now?